index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
- [ ] You did great work 😃, however, you need to fix some parts. For good syntax, Kindly make sure you follow this structure:

`````
<body>
    <main>
        <header>
               <a href="#">LOGO</a>
               <nav>
                    --- hamburger menu ---
               </nav>
        </header>
        <section>
               --- the headline section (right after the header) ----
        </section>
    </main>
</body>
`````


- [ ]  Kindly make sure your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-988&t=OmND1dx8utbvJmxn-0) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 😃,

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-05-04 110819](https://user-images.githubusercontent.com/57408419/236171109-a0595e99-4b49-432e-9015-b5fa0fbb8053.png)| |![Screenshot 2023-05-04 1](https://user-images.githubusercontent.com/57408419/236171199-b4442c00-8ec8-4a72-868c-af1da76d078e.png)| |